sys/net/transport_layer/udp/udp.h

```diff
@@ -1,16 +1,18 @@
 /**
- * UDP header
- *
- * Copyright (C) 2013  INRIA.
+ * Copyright (C) 2015  INRIA.
```

Comment:
Also, you are not working for INRIA AFAIK, so I think you should put your name/affiliation there for 2015 and just leave that line alone. Did you look into the history? Only Oliver Gesch is mentioned as an author, and he never worked for INRIA either...